Description: Use the FHS compliant path for scap's bin_dir when packaging
Author: Wikimedia Release Engineering <releng@wikimedia.org>
--- a/scap/config.py
+++ b/scap/config.py
@@ -14,7 +14,7 @@ from . import utils


 DEFAULT_CONFIG = {
-    'bin_dir': (str, '/srv/deployment/scap/scap/bin'),
+    'bin_dir': (str, '/usr/bin'),
     'canary_threshold': (float, 10.0),
     'canary_wait_time': (int, 20),
     'deploy_dir': (str, '/srv/mediawiki'),
